From 7239a8e7a9e621a093dfa0733e95a586188c10b9 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Tue, 17 May 2011 13:22:15 +0800
Subject: ENGR00143511 config: add ipu and camera module in kernel build
 config

add ipu and camera modules in kernel build configure

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   26 ++++++++++++++++++++------
 1 file changed, 20 insertions(+), 6 deletions(-)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index df5ddbd..89b7b4f 100755
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Fri Apr 22 11:17:35 2011
+# Tue May 17 09:57:25 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -254,8 +254,10 @@ CONFIG_ARCH_MXC=y
 # CONFIG_PLAT_SPEAR is not set
 # CONFIG_GPIO_PCA953X is not set
 # CONFIG_KEYBOARD_GPIO_POLLED is not set
+CONFIG_IMX_HAVE_PLATFORM_DMA=y
 CONFIG_IMX_HAVE_PLATFORM_FEC=y
 CONFIG_IMX_HAVE_PLATFORM_GPIO_KEYS=y
+CONFIG_IMX_HAVE_PLATFORM_GPMI_NFC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX2_WDT=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_SRTC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_I2C=y
@@ -1264,8 +1266,8 @@ CONFIG_REGULATOR_MC13892=y
 # CONFIG_REGULATOR_ISL6271A is not set
 # CONFIG_REGULATOR_AD5398 is not set
 # CONFIG_REGULATOR_TPS6524X is not set
-CONFIG_REGULATOR_DA9052=y
 CONFIG_REGULATOR_MAX17135=y
+CONFIG_REGULATOR_DA9052=y
 CONFIG_MEDIA_SUPPORT=y
 
 #
@@ -1399,7 +1401,21 @@ CONFIG_VIDEO_IR_I2C=y
 # CONFIG_VIDEO_UPD64031A is not set
 # CONFIG_VIDEO_UPD64083 is not set
 # CONFIG_VIDEO_VIVI is not set
-# CONFIG_VIDEO_MXC_CAMERA is not set
+CONFIG_VIDEO_MXC_CAMERA=m
+
+#
+# MXC Camera/V4L2 PRP Features support
+#
+CONFIG_VIDEO_MXC_IPU_CAMERA=y
+# CONFIG_VIDEO_MXC_CSI_CAMERA is not set
+# CONFIG_MXC_CAMERA_MICRON111 is not set
+# CONFIG_MXC_CAMERA_OV2640 is not set
+CONFIG_MXC_CAMERA_OV3640=m
+CONFIG_MXC_CAMERA_OV5640=m
+CONFIG_MXC_CAMERA_OV5642=m
+CONFIG_MXC_IPU_PRP_VF_SDC=m
+CONFIG_MXC_IPU_PRP_ENC=m
+CONFIG_MXC_IPU_CSI_ENC=m
 CONFIG_VIDEO_MXC_OUTPUT=y
 CONFIG_VIDEO_MXC_IPU_OUTPUT=y
 # CONFIG_VIDEO_MXC_IPUV1_WVGA_OUTPUT is not set
@@ -1505,9 +1521,6 @@ CONFIG_FB_MODE_HELPERS=y
 #
 # Frame buffer hardware drivers
 #
-CONFIG_FB_MXC=y
-CONFIG_FB_MXC_EINK_PANEL=y
-# CONFIG_FB_MXC_EINK_AUTO_UPDATE_MODE is not set
 # CONFIG_FB_UVESA is not set
 # CONFIG_FB_S1D13XXX is not set
 # CONFIG_FB_TMIO is not set
@@ -1942,6 +1955,7 @@ CONFIG_MXC_PXP=y
 CONFIG_MXC_PXP_CLIENT_DEVICE=y
 # CONFIG_TIMB_DMA is not set
 CONFIG_IMX_SDMA=y
+# CONFIG_MXS_DMA is not set
 CONFIG_DMA_ENGINE=y
 
 #
-- 
1.7.9.5

